src/plays/lofi-player/LofiPlayer.js

```diff
@@ -0,0 +1,65 @@
+import { useRef, useState, useEffect } from 'react';
+import PlayHeader from 'common/playlists/PlayHeader';
+
+import { SONGS_DATA } from './songs';
+import Player from './player';
+import './styles.css';
+
+function LofiPlayer(props) {
+
+  // Your Code Start below.
+  const [songs, setSongs] = useState(SONGS_DATA);
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [currentSong, setCurrentSong] = useState(SONGS_DATA[1]);
+  const handlePlayState = (value) => setIsPlaying(value);
+  const handleCurrentSong = (value) => setCurrentSong(value);
+
+  const audioElem = useRef();
+  const vidLink= "https://firebasestorage.googleapis.com/v0/b/musicplayer-bc7a2.appspot.com/o/video%2F7btrrd%20(1).mp4?alt=media&token=4a59fcc2-368b-40b4-a486-6cbff3c5c98b";
+
+  useEffect(() => {
+    if (isPlaying) {
+      audioElem.current.play();
+    }
+    else {
+      audioElem.current.pause();
+    }
+  }, [isPlaying])
+
+  const handleOnPlay = () => {
+    const duration = audioElem.current.duration;
+    const ct = audioElem.current.currentTime;
+
+    setCurrentSong({ ...currentSong, "progress": ct / duration * 100, "length": duration })
+  }
+  
+
+  return (
+    <>
+
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className='lofi-all'>
```

Comment:
@riyajath-ahamed The problem is this structure is wrong!

Ideally all these should come inside 

```
<div className="play-details-body">
```

You can see it results in responsiveness

![image](https://user-images.githubusercontent.com/3633137/198818728-184e2663-c1f8-4d53-8ad9-fb6515b6ef4d.png)

Please fix it, I want to merge it asap.

Reply:
@riyajath-ahamed Any update?

Reply:
Sorry for the delay. I will fix that.

Reply:
Thanks @riyajath-ahamed . Have you fixed it?

Reply:
Yes fixed.

Reply:
@riyajath-ahamed I think you have removed the BG image now!

However I still see a small responsiveness issue

<img width="421" alt="image" src="https://user-images.githubusercontent.com/3633137/201593918-16f638cc-af1d-4ead-a0b5-d7f236a13000.png">

Reply:
@riyajath-ahamed Did you get a chance to look into this comment?